lib/plugins/bugsnagPlugin.ts

```diff
@@ -4,12 +4,13 @@ import {
   bugsnagErrorReporter,
   type ErrorReport,
   reportErrorToBugsnag,
+  addFeatureFlag,
   type Severity,
 } from '@lokalise/error-utils'
 import type { FastifyInstance } from 'fastify'
 import fp from 'fastify-plugin'
 
-export { reportErrorToBugsnag, bugsnagErrorReporter, ErrorReport, Severity }
+export { reportErrorToBugsnag, addFeatureFlag, bugsnagErrorReporter, ErrorReport, Severity }
```

Comment:
you also need to update export at https://github.com/lokalise/fastify-extras/blob/main/lib/index.ts

Reply:
Good cach :sunglasses: